crates/aggchain-proof-program/Cargo.toml

```diff
@@ -1,10 +1,10 @@
 [package]
 version = "0.1.0"
-name = "sp1-ecdsa-verification-program"
+name = "aggchain-proof-program"
 edition = "2021"
 
 [dependencies]
-alloy-primitives.workspace = true
+alloy-primitives = { version = "0.8.15", features = ["serde", "k256"] }
```

Comment:
`alloy` version `0.8` is a bit behind, why couldn't we use workspace version `0.11` here?